install.sh

```diff
@@ -151,12 +160,49 @@ install_node() {
     fi
 }
 
+snapshot_restore() {
+    if [ $SHOW_PROMPT = true ]; then
+        echo -e "\nATTENTION: This script will restore the node database from a snapshot.\n"
```

Comment:
We should also warn the user that their database dir will be deleted and restored (overwritten) by the snapshot.

Reply:
c8bad5e